02-oop-inheritance/src/classes/ActorClass.js

```diff
@@ -0,0 +1,6 @@
+export default class Actor {
```

Comment:
Is not needed to call the file `ActorClass` if you keep consistent when using names for variables (only use upper case first letter for Classes) this is understanded by default